pkg/tsdb/sqleng/sql_engine.go

```diff
@@ -29,7 +31,7 @@ var XormDriverMu sync.RWMutex
 // MetaKeyExecutedQueryString is the key where the executed query should get stored
 const MetaKeyExecutedQueryString = "executedQueryString"
 
-var ErrConnectionFailed = errors.New("failed to connect to server - please inspect Grafana server log for details")
+var ErrConnectionFailed = errutil.NewBase(errutil.StatusInternal, "sql.connectionError")
```

Comment:
Please include this suggestion if you agree with it. I'm changing the service name to match the root service directory.

```suggestion
var ErrConnectionFailed = errutil.NewBase(errutil.StatusInternal, "sqleng.connectionError")
```

Reply:
accepted, thanks!